tests/behat/features/capabilities/event/event-create.feature

```diff
@@ -60,3 +60,83 @@ Feature: Create Event
     Then I should see "Access denied"
       And I should see "You are not authorized to access this page."
       And I enable that the registered users to be verified immediately
+
+  @verified @perfect @critical
```

Comment:
Same as the above goes here probably, also I'm not sure if the issue is in the event create range or if it's just the viewing that goes wrong, probably the latter and in that case you don't have to change the event-create test but can rather use different events above piped together, like such:

```
    Given events:
      | title          | author      | field_event_date    | body                   | field_content_visibility | langcode |
      | Event content 1| ExportUser1 | 2100-01-01T12:00:00 | Body description text 1| public                   | en       |
      | Event content 2| ExportUser2 | 2100-01-01T12:00:00 | Body description text 2| public                   | en       |
      | Event content 3| ExportUser3 | 2100-01-01T12:00:00 | Body description text 3| public                   | en       |
```